### PR TITLE
Use the gen_event behaviour insteed of GenEvent

### PR DESCRIPTION
**Pull request by glejeune**
GenEvent is deprecated

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -25,7 +25,7 @@ defmodule LoggerLagerBackend.Mixfile do
 
   defp deps do
     [
-      {:lager, "~> 3.2"},
+      {:lager, ">= 3.2.0 and < 4.0.0"},
```
Comment (jonathanperret) [inline mix.exs]: According to https://hexdocs.pm/elixir/Version.html#module-requirements, this is exactly equivalent to the original expression (`~> 3.2`). Any reason to change this?

**Comment by glejeune** [comment]
Any chance to merge or not ?

**Comment by jonathanperret** [comment]
Of course! Thanks and sorry for the lag.

**Comment by glejeune** [comment]
@jonathanperret :pray:

**Comment by jonathanperret** [comment]
Included in 0.2.0. Sorry (again!) for the delay.

I took out the lager dependency bump though, it did not seem required unless I missed something?

**Pull request merged**